llvm/test/SYCLLowerIR/ConditionalCallOnDevice/conditional-call-on-device-1.ll

```diff
@@ -0,0 +1,40 @@
+; RUN: opt -passes=sycl-conditional-call-on-device -sycl-conditional-call-on-device-unique-prefix="PREFIX" < %s -S | FileCheck %s
+
+%class.anon = type { ptr addrspace(4) }
+%"struct.std::integer_sequence.3" = type { i8 }
+
+define internal spir_func void @_ZN4sycl3_V13ext6oneapi12experimental6detail38call_if_on_device_conditionally_helper(ptr noundef byval(%class.anon) align 8 %fn, ptr noundef byval(%"struct.std::integer_sequence.3") align 1 %0) #2 !srcloc !0 {
```

Comment:
Considering that you now using an attribute to detect those helpers, you can now rename them here to further improve tests readability.